src/main/resources/application.yml

```diff
@@ -52,6 +52,7 @@ application:
       placement: ${PLACEMENT_QUEUE_URL:}
       placement-specialty: ${PLACEMENT_SPECIALTY_QUEUE_URL:}
       post: ${POST_QUEUE_URL:}
+      post-specialty: ${POST_SPECIALTY_QUEUE_URL:}
```

Comment:
There is no matching ECS task template change to allow this to be set on deploy.

Reply:
Just, uh... making sure to keep you on your toes :trollface: . Thanks @Judge40 - there is now a PR for the queues as well https://github.com/Health-Education-England/TIS-OPS/pull/657